src/screens/OrganizationDashboard/OrganizationDashboardMocks.ts

```diff
@@ -298,3 +299,23 @@ export const ERROR_MOCKS = [
     error: new Error('Mock Graphql ORGANIZATION_EVENT_LIST Error'),
   },
 ];
+
+export const mockEventData = {
```

Comment:
Remove `mockEventData` from here. We are using this only in a single test. Declare the variable inside the test itself (i.e. inside this test `Testing useEffect hook and error redirection`)

Reply:
Done please Check